CHANGES.rst

```diff
@@ -13,6 +13,17 @@ Changelog
 
 .. towncrier release notes start
 
+1.4.0 (2021-06-30)
```

Comment:
This needs the same fix as https://github.com/pulp/pulp-certguard/pull/105/commits/12a26ea46faa185e333895c07a1c47559431b344